.github/workflows/track_minetest.yml

```diff
@@ -23,5 +30,61 @@ jobs:
           git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
           ./update-revs.sh
           ./push-revs.sh
```

Comment:
You can still have it stop here if the revisions haven't changed

Reply:
You mean by generating an error? I was hoping to avoid making the benchmark build continuously red. Also there's an upside to doing extra runs, it helps to even out outliers in the graph so you can see where the average is:

https://paradust7.github.io/benchmarks/dev/bench/

If we keep 1 month of history, for a run every 3 hours, that's ~240 data points per graph.

Reply:
I added a skip, tested it, works.